ci-operator/step-registry/hypershift/mce/ibmz/create/hcp/hypershift-mce-ibmz-create-hcp-commands.sh

```diff
@@ -143,7 +143,12 @@ echo "$(date) Creating agent hosted cluster manifests"
 oc create ns ${hcp_ns}
 mkdir /tmp/hc-manifests
 
-${HYPERSHIFT_CLI_NAME} create cluster agent \
+ICSP_COMMAND=""
+if [[ $ENABLE_ICSP == "true" ]]; then
+  ICSP_COMMAND=$(echo "--image-content-sources ${SHARED_DIR}/mgmt_iscp.yaml")
```

Comment:
@heliubj18 , this yaml will be generated once the mgmt cluster is created ?

Is this file name `mgmt_iscp.yaml` only or is it `mgmt_icsp.yaml` ? Just for the correct file name confirmation.

Reply:
currently, it is `iscp.yaml` .. It is a typo here. I need to submit another pr to update it because there are multiple steps/chains using this one now.